catalog/data_volume.go

```diff
@@ -0,0 +1,50 @@
+package catalog
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/service/catalog"
+	"github.com/databricks/terraform-provider-databricks/common"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type volumeDataParams struct {
+	FullName    string `json:"full_name,omitempty"`
+	CatalogName string `json:"catalog_name,omitempty"`
+	SchemaName  string `json:"schema_name,omitempty"`
+	Name        string `json:"name,omitempty"`
+}
+
+func (volumeDataParams) Aliases() map[string]string {
```

Comment:
the `Aliases` and `CustomizeSchema` func are not applied in `genericDatabricksData`, which leads to tests failure